### DISCUSSION TITLE
Welcome to clipio Discussions!

### DISCUSSION THREAD

**Discussion by pTinosq** [pTinosq/clipdit#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by orattray** [comment]
Hello there!

I was just wondering what your timeline is for a macos release.

**Reply by pTinosq** [reply]
Hello Orattray,
There are no current plans for a MacOS release due to limited resources.

If you would like to contribute to the project, feel free to - we're always looking for contributors :)
